calculator.js

```diff
@@ -191,16 +192,37 @@ function calculateResult(operator, num1, num2){
         case '-':
             result = subtract(num1, num2);
             break;
-    }
-    if (result.toString().length > 10){
-        console.log('long number');
-        result = result.toFixed(9);
-    }
-    digitsOnScreen.innerHTML = result;
-    console.log(result);
+    }    
+
     return result;
 }
 
+function doAfterGettingResult(result, opType){
```

Comment:
The name of the function is not really saying what it is doing, but stating that the developer want to use this function after get the result.

The same suggestion made in the `showOnScreen` is applicable here.

Notice that the `showOnScreen` is not passing anything, making this call inconsistent with the other calls in the code.

Reply:
got rid of the function after realizing that a good chunk of the code could be (and should be) performed by handlesReset()